Use void so that clang does not complain about missing (unused)
return value, and also add missing string.h.
--- a/src/ranking.c
+++ b/src/ranking.c
@@ -1,2 +1,3 @@
 #include <pwd.h>
+#include <string.h>
 #include "geki4.h"
@@ -7,3 +8,3 @@
  **********************/
-int ScoreRanking(void)
+void ScoreRanking(void)
 {
--- a/src/ranking.h
+++ b/src/ranking.h
@@ -3,2 +3,3 @@
 
+void ScoreRanking(void);
 void RankingScore(void);
